rlcard/games/nolimitholdem/game.py

```diff
@@ -113,7 +113,7 @@ def get_legal_actions(self):
         """
         return self.round.get_nolimit_legal_actions(players=self.players)
 
-    def step(self, action):
+    def step(self, action_tp):
```

Comment:
What is the motivation of using action_tp? What would the legal actions be with action_tp?

Reply:
action_tp (or action_tuple) [Action, amt: int]
legal actions will only contain FOLD, RAISE, CALL,
In the case of raise, action_tp[1] indicates the amt to raise; In other cases, action_tp[1] is not used